controllers/drcluster_mmode.go

```diff
@@ -59,6 +61,10 @@ func (u *drclusterInstance) mModeActivationsRequired() (map[string]ramen.Storage
 	}
 
 	for _, drpcCollection := range drpcCollections {
+		if !drpcCollection.drpc.GetDeletionTimestamp().IsZero() {
+			continue
+		}
```

Comment:
Looks good, but `!drpcCollection.drpc.GetDeletionTimestamp().IsZero()` is not clear. It will be much easier to understand this if we add a helper like drClsuterIsDeleted(). We can actually add generic helper since all resources support GetDeletionTimestamp().

Reply:
@nirs I added an issue to create generic helper for all resources: https://github.com/RamenDR/ramen/issues/1176